src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+Example .. code-block:: html
+        
+         <ad:priceType>FIXED_PRICE</ad:priceType>
+======= ============================================
+
+.. index:: price
+.. _feed_price:
+
+price
+"""""
+
+The meaning of the value depends on the :ref:`feed_priceType`. 
+
+If it is `FIXED_PRICE` or `BIDDING_FROM` then **price** is mandatory and needs to be greater than 0.
+The maximum allowed **price** value is ``10000000000`` of the local market currency. (100.000.000,00 EUR / CAD / ... ).
+
+======= =============================
+Example .. code-block:: html
+        
+         <ad:price>1500</ad:price>
+======= =============================
+
+.. index:: originalPrice
+.. _feed_originalPrice:
+
+originalPrice
+""""""""""""""
+
+Product price before discount. Ignored if a seller does not have a discount option enabled. Must be greater than :ref:`feed_price`.
+The maximum allowed **originalPrice** value is ``10000000000`` of the local market currency. (100.000.000,00 EUR / CAD / ... ).
+
+======= ============================================
+Example .. code-block:: html
+        
+         <ad:originalPrice>1500</ad:originalPrice>
+======= ============================================
+
+.. index:: media
+.. _feed_media:
+
+media
+"""""
+
+Product images. Media should contain from 0 to N **<image>** elements, where the exact limit depends on the category in taxonomy. **<image>** elements must contain an URL attribute. URL should be a complete link pointing to an image on a publicly available webserver.
+
+Allowed image formats: JPEG, JPG, PNG, GIF\*, BMP.
+
+\* Please note that GIFs are not recommended format as thay are only 256 colors or less.
+
+All images will be resized if necessary to a size of maximum 1024px height and 1024px width (preserving the aspect ratio)
+The system will download the images and, if they meet the requirements, store them on our servers in several sizes.
+
+The images will be presented in the order as provided. The first image is shown in search results and as the main image on the item page.
+
+======= ========================================================================================================================
+Example .. code-block:: html 
+
+            <ad:media>
+                <ad:image url="https://images.pexels.com/photos/62289/yemen-chameleon-chamaeleo-calyptratus-chameleon-reptile-62289.jpeg"/>
+                <ad:image url="https://images.pexels.com/photos/47547/squirrel-animal-cute-rodents-47547.jpeg?auto=compress&amp;cs=tinysrgb&amp;w=1260&amp;h=750&amp;dpr=2"/>
+            <ad:media/>
+======= ========================================================================================================================
+
+.. index:: attributes
+.. _feed_attr:
+
+attributes
+""""""""""
+
+Optional collection of ad :ref:`user_defined_attributes` (category-dependent). 
+Note that categories can have mandatory attributes, for which a default value will be filled in if not supplied.
```

Comment:
are you sure this is _always_ the case, that there is a default value? do we never invalidate if a mandatory attribute is not supplied?

Reply:
Maybe we can add what @hallidayk mentioned here: these can be used to influence the ad relevance.

Reply:
@pmongh worth checking the first comment as well, just in case. We saw from the API examples that somehow we act differently in the feed world than what happens down the road in ad service.

Reply:
This will protect us from any "guarantees" that the users might take for granted at this stage, so I'd add it it

Reply:
removed